features/informatievragen-fields.feature

```diff
@@ -0,0 +1,621 @@
+      | verblijfadres.huisnummer          | <huisnummer>                       |
+      | verblijfadres.postcode            | <postcode>                         |
+      | verblijfadres.woonplaats          | <woonplaats>                       |
+      | verblijfadres.locatiebeschrijving | <locatiebeschrijving>              |
+      | verblijfadres.land.code           | <land>                             |
+      | verblijfadres.land.omschrijving   | <omschrijving>                     |
+      | verblijfadres.regel1              | <regel1>                           |
+      | verblijfadres.regel2              | <regel2>                           |
+      | verblijfadres.regel3              | <regel3>                           |
+      | adresseerbaarObjectIdentificatie  | <adresseerbaarObjectIdentificatie> |
+      | nummeraanduidingIdentificatie     | <nummeraanduidingIdentificatie>    |
+      | datumAanvangAdreshouding          | <datumAanvangAdreshouding>         |
+      | datumAanvangAdresBuitenland       | <datumAanvangAdresBuitenland>      |
+
+      Voorbeelden:
+      | burgerservicenummer | functieAdres | functieAdres omschrijving | datumAanvangAdreshouding | straat    | huisnummer | postcode | woonplaats | adresseerbaarObjectIdentificatie | nummeraanduidingIdentificatie | locatiebeschrijving        | land | land omschrijving | datumAanvangAdresBuitenland | regel1            | regel2       | regel 3 |
+      | 999993653           | W            | woonadres                 | 20150808                 | Boterdiep | 31         | 3077AW   | Rotterdam  | 0599010000208579                 | 0599200000219678              |                            |      |                   |                             |                   |              |         |
+      | 000009921           |              |                           |                          |           |            |          |            |                                  |                               | Woonboot in de Grote Sloot |      |                   |                             | Locatie           |              |         |
+      | 999995121           |              |                           |                          |           |            |          |            |                                  |                               |                            | 5002 | Frankrijk         |                             | 14 Rue Camulogene | 75015 Parijs |         |
+      | 999993586           |              |                           |                          |           |            |          |            |                                  |                               |                            | 0000 | Onbekend          |                             |                   |              |         |
+
+  Rule: wanneer gevraagd wordt om vanuitVerblijfplaatsOnbekend, wordt landVanwaarIngeschreven geleverd
```

Comment:
Hier zit voor mijn gevoel iets niet goed in de specificatie maar ook niet in deze vraag.
* Ik zou verwachten dat ik alleen `vanuitVerblijfplaatsOnbekend` zou krijgen als er geen verblijfplaats bekend is waarvanuit de persoon geimmigreerd is. Missen we in `immigratie` dus niet de velden om die verblijfplaats wel te specificeren.
* Waarom zou je `vanuitVerblijfplaatsOnbekend` überhaupt opvragen. Mij lijkt dat je die nooit opvraagt maar automatisch moet krijgen als er geen verblijfplaats bekend is waarvanuit geimmigreerd is.

Reply:
bedoel je automatisch krijgen wanneer iets van immigratie gevraagd is of echt altijd automatisch? dat laatste lijkt me niet de bedoeling. lang niet elke gebruiken mag iets weten over immigratie (is een discriminatie gevoelig gegeven).

automatisch leveren van vanuitVerblijfplaatsOnbekend na vragen landVanwaarIngeschreven staat m.i. los van het alleen kunnen vragen van vanuitVerblijfplaatsOnbekend. Er zijn situaties waarbij de gebruiker geen doelbinding heeft voor immigratieland of immigratiedatum, maar wel voor vanuitVerblijfplaatsOnbekend.

Reply:
besproken. het algoritme voor het bepalen van vanuitVerblijfplaatsOnbekend houden we nu zoals gespecificeerd in de feature. Kunnen we op een later moment aanpassen.